File: bots_templates/8_claude_ai.py
# ====================================================
# بوت Claude AI - محول من PHP إلى Python
# ====================================================

import telebot
import requests
from telebot import types

TOKEN = "{TOKEN}"
OWNER_ID = {OWNER_ID}
CHANNEL = "@Namiro_1"
API_URL = "https://devil.xo.je/v/ai/claude.php"

bot = telebot.TeleBot(TOKEN)

def ask_claude(message_text):
    try:
        resp = requests.get(API_URL, params={"message": message_text}, timeout=60)
        if resp.status_code == 200:
            data = resp.json()
            if data.get("success"):
                return data.get("response", "عذراً، لم أستطع معالجة طلبك.")
            elif data.get("response"):
                return data["response"]
            elif data.get("error"):
                return f"⚠️ خطأ: {data['error']}"
    except Exception as e:
        return f"⚠️ خطأ في الاتصال: {str(e)}"
    return "⚠️ عذراً، حدث خطأ. حاول مرة أخرى."

@bot.message_handler(commands=["start"])
def start(msg):
    kb = types.InlineKeyboardMarkup()
    kb.add(types.InlineKeyboardButton("📢 قناة البوت", url=f"https://t.me/{CHANNEL.replace('@','')}"))
    bot.send_message(
        msg.chat.id,
        f"*مرحباً بك في بوت المحادثة بالذكاء الاصطناعي!*\n\n"
        f"أنا بوت يعمل بنموذج *Claude AI*\n\n"
        f"📌 *الطريقة:*\n"
        f"فقط أرسل سؤالك وسأجيبك فوراً.\n\n"
        f"📢 *قناة البوت:* {CHANNEL}",
        parse_mode="Markdown",
        reply_markup=kb
    )

@bot.message_handler(func=lambda m: m.text and not m.text.startswith("/"))
def handle_message(msg):
    bot.send_chat_action(msg.chat.id, "typing")
    reply = ask_claude(msg.text)
    bot.send_message(msg.chat.id, reply, parse_mode="Markdown")

if __name__ == "__main__":
    print("🤖 Claude AI Bot Started...")
    bot.infinity_polling(timeout=60, long_polling_timeout=60)

File: bots_templates/8_claude_ai.php
<?php
// 
//حقوق @ROIUCX
//حقوق @ROIUCX

$token   = "TOKEN";  // ضع توكن حب 
$api_url = "https://devil.xo.je/v/ai/claude.php";  //   
$channel = "@ROIUCX";      

$update = json_decode(file_get_contents("php://input"), true);

//حقوق @ROIUCX
//حقوق @ROIUCX
if (isset($update['message'])) {
    $message = $update['message'];
    $chat_id = $message['chat']['id'];
    $text = trim($message['text'] ?? '');
    
    //حقوق @ROIUCX
//حقوق @ROIUCX
    if ($text == '/start') {
        $welcome = " *مرحباً بك في بوت المحادثة بالذكاء الاصطناعي!*\n\n";
        $welcome .= "أنا بوت يعمل بنموذج *Claude 3.7 Sonnet*\n\n";
        $welcome .= " *الطريقة:*\n";
        $welcome .= "فقط أرسل سؤالك أو رسالتك وسأجيبك فوراً.\n\n";
        $welcome .= " *قناة البوت:* $channel";
        
        //حقوق @ROIUCX
//حقوق @ROIUCX
        $keyboard = json_encode([
            'inline_keyboard' => [
                [['text' => ' قناة البوت', 'url' => 'https://t.me/ROIUCX']]
            ]
        ]);
        
        sendMessage($token, $chat_id, $welcome, $keyboard);
        exit;
    }
    
   //حقوق @ROIUCX
//حقوق @ROIUCX
    if (!empty($text) && !str_starts_with($text, '/')) {
     //حقوق @ROIUCX
//حقوق @ROIUCX
        file_get_contents("https://api.telegram.org/bot{$token}/sendChatAction?chat_id={$chat_id}&action=typing");
        
     //حقوق @ROIUCX
//حقوق @ROIUCX
        $response = callClaudeAPI($api_url, $text);
        
        if ($response && isset($response['success']) && $response['success'] == true) {
            $reply = $response['response'] ?? 'عذراً، لم أستطع معالجة طلبك.';
            sendMessage($token, $chat_id, $reply);
        } elseif ($response && isset($response['response'])) {
            sendMessage($token, $chat_id, $response['response']);
        } elseif ($response && isset($response['error'])) {
            sendMessage($token, $chat_id, " *خطأ:*\n" . $response['error']);
        } else {
            sendMessage($token, $chat_id, " عذراً، حدث خطأ في الاتصال. حاول مرة أخرى.");
        }
    }
}

//حقوق @ROIUCX
//حقوق @ROIUCX

function callClaudeAPI($api_url, $message) {
    $ch = curl_init();
    curl_setopt($ch, CURLOPT_URL, $api_url . "?message=" . urlencode($message));
    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
    curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
    curl_setopt($ch, CURLOPT_TIMEOUT, 60);
    
    $response = curl_exec($ch);
    $httpCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
    curl_close($ch);
    
    if ($httpCode != 200 || !$response) {
        return null;
    }
    
    return json_decode($response, true);
}

function sendMessage($token, $chat_id, $text, $keyboard = null) {
    $url = "https://api.telegram.org/bot{$token}/sendMessage";
    $postData = [
        'chat_id' => $chat_id,
        'text' => $text,
        'parse_mode' => 'Markdown'
    ];
    if ($keyboard) {
        $postData['reply_markup'] = $keyboard;
    }
    
    $ch = curl_init();
    curl_setopt($ch, CURLOPT_URL, $url);
    curl_setopt($ch, CURLOPT_POST, true);
    curl_setopt($ch, CURLOPT_POSTFIELDS, $postData);
    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
    curl_exec($ch);
    curl_close($ch);
}
?>